introduction.md

```diff
@@ -1,2 +1,7 @@
 I'm learning to use git!
 I love the JavaScript path.
+*Git
+*JavaScript
+*HTML
+*CSS
+*Java
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?